README.adoc

```diff
@@ -28,8 +60,9 @@ endif::[]
 
 == Acknowledgements
 
-* Some parts of this sample application were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
+* Some parts of Organice were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
 _Marco Jakob_.
 * Libraries used: https://openjfx.io/[JavaFX], https://github.com/FasterXML/jackson[Jackson], https://github.com/junit-team/junit5[JUnit5]
+* Organice is adapted from AddressBook-Level3 project created by SE-EDU initiative at https://se-education.org
```

Comment:
Suggested change:
Organice is adapted from AddressBook-Level3, a project created by the SE-EDU initiative.
The 'SE-EDU initiative' could include a hyperlink to their website.